designdocs.md

```diff
@@ -79,6 +80,19 @@ Obviously, we need specs too. But they often come later in the lifecycle, and ar
 
 We could capture a designdoc in a github issue, and this may in fact be a good convention. But a typical issue or bug has different content than what goes into a designdoc and sometimes you don’t want to parse all the discussion, you just want the conclusion.
 
+## Design Doc Process
+* Create new docs in the `Filecoin Community` > `Design Docs` folder.
+  * Check that you're using the GApps identity
```

Comment:
I'm not 100% I know what this means.  Is there a separate `GApps` id I need to log into to do this?  Is this suggesting to sign in with your own appropriately chosen id?

Reply:
"If you have one, check that ..."

Reply:
Added “...identity of your choice”